pkg/ccr/handle/rename_partition.go

```diff
@@ -0,0 +1,60 @@
+package handle
+
+import (
+	"github.com/cloudwego/kitex/tool/internal_pkg/log"
+	"github.com/selectdb/ccr_syncer/pkg/ccr"
+	"github.com/selectdb/ccr_syncer/pkg/ccr/record"
+	festruct "github.com/selectdb/ccr_syncer/pkg/rpc/kitex_gen/frontendservice"
+)
+
+func init() {
+	ccr.RegisterJobHandle[*record.RenamePartition](festruct.TBinlogType_RENAME_PARTITION, &RenamePartitionHandle{})
+}
+
+type RenamePartitionHandle struct {
+}
+
+func (h *RenamePartitionHandle) IsBinlogCommitted(j *ccr.Job, r *record.RenamePartition) (bool, error) {
+	_, err := j.GetDestNameBySrcId(r.TableId)
+	if err != nil {
+		return false, err
+	}
+
+	partitionName, err := j.GetDestMeta().GetPartitionName(r.TableId, r.PartitionId)
+	if err != nil {
+		return false, err
+	}
+
+	if partitionName == r.OldPartitionName {
+		return false, nil
+	}
+
+	return true, nil
+}
```

Comment:
![image](https://github.com/user-attachments/assets/c6ba16a6-c58f-411c-a50f-1dd221e98967)
Please follow the original implementation.